DEVELOPMENT.md

```diff
@@ -151,6 +151,84 @@ to start a debugger to which attach from your IDE.
    curl localhost:8081/apis/apps/v1/deployments
    ```
 
+## Database
+
+### Database requirements
+
+* psql
+```bash
+$ sudo dnf install postgresql
+$ psql --version
+psql (PostgreSQL) 15.6
+```
+* A SQL client is recommended to interact easily with the database. For example [DBeaver Community](https://dbeaver.io/).
+
```

Comment:
This is not needed for development, if someone prefers to use a SQL client of install `psql` they can do it on their own, however you can access the database running psql on the database container.

Reply:
Modified to
/* Optional: A SQL client is recommended to interact easily with the database. For example [DBeaver Community](https://dbeaver.io/). To configure DBeaver, check these [instructions](https://github.com/kubearchive/kubearchive/blob/main/database/README.md).
/

Reply:
This reminds me of the provided suggestion of using podman desktop to manage the local kubernetes cluster with kind. I moved it to the `Tools` Section in the `DEVELOPER_RESOURCES.md` file. We can do the same with DBeaver, and maybe just have a pointer to the `DEVELOPER_RESOURCES.md` file in this point.

Reply:
Also, I think @rh-hemartin also refers to the first part of this requirements where `psql` is installed. It probably shouldn't be in the requirements as the developer can use the `psql` command inside the container

Reply:
And they don't need to run the psql tool either, as the script interacts with the database for them.

Reply:
moved